Classes/Accounts/Detail/AccountInformation/StandardAccountInformation/StandardAccountInformationScreen.swift

```diff
@@ -133,9 +133,8 @@ extension StandardAccountInformationScreen {
             $0.trailing == 0
         }
 
-        accountTypeInformationView.startObserving(event: .performHyperlinkAction) {
-            [unowned self] in
-            self.open(AlgorandWeb.rekey.link)
+        accountTypeInformationView.startObserving(event: .performHyperlinkAction) { [unowned self] in
```

Comment:
Try to use `weak` instead `unowned` when possible. Unowned references can become dandling pointers under some conditions.